NMS/v1_12_R1/src/main/java/com/bgsoftware/superiorskyblock/nms/v1_12_R1/NMSPlayersImpl.java

```diff
@@ -40,12 +40,27 @@ public NMSPlayersImpl(SuperiorSkyblockPlugin plugin) {
         this.plugin = plugin;
     }
 
+    @Override
+    public void clearEnderChest(OfflinePlayer offlinePlayer) {
+        clearInventoryInternal(offlinePlayer, false, true);
+    }
+
     @Override
     public void clearInventory(OfflinePlayer offlinePlayer) {
+        clearInventoryInternal(offlinePlayer, true, false);
+    }
+
+    public void clearInventoryInternal(OfflinePlayer offlinePlayer, boolean inventory, boolean enderchest) {
```

Comment:
I'd expose the clearInventoryInternal method to NMSPlayers interface instead of seperating it into two different methods.
This is because when you want to clear both inventory and ender chest, you won't need to call this function twice (which loads data from disk twice for the player - expensive!)